qiita_pet/templates/edit_study.html

```diff
@@ -20,168 +20,71 @@
 </style>
 
 <script>
-// qutocomplete combobox code modified from http://jqueryui.com/autocomplete/#combobox, retrieved 8 Sept 2014
-(function( $ ) {
-    $.widget( "custom.combobox", {
-      _create: function() {
-        this.wrapper = $( "<span>" )
-          .addClass( "custom-combobox" )
-          .insertAfter( this.element );
-
-        this.element.hide();
-        this._createAutocomplete();
-        this._createShowAllButton();
-      },
-
-      _createAutocomplete: function() {
-        var selected = this.element.children( ":selected" ),
-          value = selected.val() ? selected.text() : "";
-
-        this.input = $( "<input>" )
-          .appendTo( this.wrapper )
-          .val( value )
-          .attr( "title", "" )
-          .addClass( "custom-combobox-input ui-widget ui-widget-content ui-state-default ui-corner-left" )
-          .autocomplete({
-            delay: 0,
-            minLength: 0,
-            source: $.proxy( this, "_source" )
-          })
-          .tooltip({
-            tooltipClass: "ui-state-highlight"
-          });
-
-        this._on( this.input, {
-          autocompleteselect: function( event, ui ) {
-            ui.item.option.selected = true;
-            this._trigger( "select", event, {
-              item: ui.item.option
-            });
-          },
-
-          autocompletechange: "_removeIfInvalid"
+$(document).ready(function() {
+    $("#new_person_div").hide();
+     $( "#{{creation_form.principal_investigator.id}}" ).chosen({
+        allow_single_deselect: true,
```

Comment:
Just FYI, the motivation to remove this is to keep (as much as possible) a homogeneous code-base and I pointed this out in case you missed it or added it by accident since this is not done in a consistent way throughout this PR or anywhere else (that I remember).

I wouldn't mind merging as is (granted that someone else decides this should be overlooked), but then I'm unsure what should I highlight as part of the code review and what I shouldn't. Your point is indeed true that there's many other things where time could be better rather than discussing this, and in that light I would like to not discuss this any further.